docs/docs/user-guide/get-started/deploying-configurations.md

```diff
@@ -60,9 +60,9 @@ Executing the cmdlet will automatically authenticate against the affected worklo
   <figcaption>Initiating a MOF deployment</figcaption>
 </figure>
 
-It is normal for this process to take several minutes if not hours to complete, based on how many components are defined in your configuration. It is important to understand that once the configuration completes its deployment, this will configure the PowerShell DSC engine on the current system to perform frequent checks against your Microsoft 365 tenant to check for configuration drifts. By default, the engine will wake up every 15 minutes (minimum value possible). For more details on how to configure this, please refer to <a href="https://docs.microsoft.com/en-us/powershell/dsc/managing-nodes/metaconfig?view=dsc-1.1" target="_blank">Configuring the Local Configuration Manager</a>.
+It is normal for this process to take several minutes (if not hours) to complete, based on how many components are defined in your configuration. It is important to understand that once the configuration completes its deployment, this will configure the PowerShell DSC engine (LCM) on the current system to perform frequent repeated  checks against your Microsoft 365 tenant to test for configuration drifts. By default, the engine will wake up every 15 minutes (the minimum value possible). For more details on how to configure this, please refer to <a href="https://docs.microsoft.com/en-us/powershell/dsc/managing-nodes/metaconfig?view=dsc-1.1" target="_blank">Configuring the Local Configuration Manager</a>.
```

Comment:
Just one small thing: The abbreviation LCM is not explained before it is used. Local Configuration Manager (LCM) is explained within the hyper link later on

Reply:
Good catch and completely agree